vowpalwabbit/cbify.cc

```diff
@@ -343,7 +343,7 @@ void predict_or_learn(cbify& data, single_learner& base, example& ec)
   if (use_cs)
     csl = std::move(ec.l.cs);
   else
-    ld = std::move(ec.l.multi);
+    ld = ec.l.multi;
```

Comment:
i guess we don't foresee any changes where this would make a difference?

Reply:
This resolves the lint of moving a trivially copyable type having no effect. If this object becomes non-trivial where the move would be necessary then this CI will tell us because of the `performance-unnecessary-copy-initialization` check.